gui.py

```diff
@@ -4,13 +4,13 @@
 """
 
 import os
-import datetime
 import dash_core_components as dcc
 import dash_html_components as html
 import dash_dangerously_set_inner_html as ddsih
 import luts
 import data
 import charts
+from datetime import datetime
```

Comment:
Move this back to line 7 and run pylint; a standard library import (like datetime) should come before 3rd part imports, which come before local imports. (C0411 is the error).

Reply:
I have reordered this according to Pylint and pushed the change to the PR.